pkg/spec/spec.go

```diff
@@ -315,6 +315,14 @@ func (config *CreateConfig) createConfigToOCISpec(runtime *libpod.Runtime, userM
 	}
 	configSpec := g.Config
 
+	var capRequired []string
+	for key, val := range config.Labels {
+		if key == "io.containers.capabilities" {
```

Comment:
This should really be a constant.

Reply:
I would love the constant to be in `pkg/capabilities` as we do for the seccomp policies now in [pkg/seccomp](https://github.com/containers/libpod/blob/master/pkg/seccomp/seccomp.go).

Reply:
Will have to move this to containers/common since pkg/capabilities is in there now, Also I think this should be a list. since it might change in the future if this got standardized and they picked a different name.